### PR TITLE
Fix opening of links in new tab using ctrl.

### PR DESCRIPTION
**Pull request by knight-bubble**
#2858

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://hirtie-maxim-fix-19639-page-reloading-2.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://hirtie-maxim-fix-19639-page-reloading-2.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

**Comment by 0xF013** [comment]
checked shift, ctrl and cmd in Chrome and Firefox (osx) and in some IE, Firefox (Windows).
:+1:

File: frontend/app/work_packages/directives/work-packages-table-directive.js [hunk 1]
```diff
@@ -207,17 +207,23 @@ module.exports = function(I18n, WorkPackagesTableService, $window, $timeout, fla
...
+            return;
+          }
+
           if ($event.shiftKey) {
             clearSelection();
             activeSelectionBorderIndex = WorkPackagesTableService.selectRowRange(scope.rows, row, activeSelectionBorderIndex);
           } else if($event.ctrlKey){
             setRowSelectionState(row, multipleChecked ? true : !currentRowCheckState);
-          } else {
+          }
```
Comment (myabc) [inline frontend/app/work_packages/directives/work-packages-table-directive.js]: @hirtie-maxim no need for this formatting change

**Comment by myabc** [comment]
@hirtie-maxim please also re-enable the Protractor spec that was disabled in ef8491e11b37f1e682c71906557c20c43cab867d

**Comment by knight-bubble** [comment]
Removed spec that checks is opened window, by click or not. This test will never be green on travis, cause it doesn't have windows :smile:

**Comment by 0xF013** [comment]
@hirtie-maxim please remove WIP label when it's ready to be merged

**Pull request merged**